sectionsUtilityTool/src/main/webapp/assets/js/utils.js

```diff
@@ -11,7 +11,10 @@ $.ajaxSetup({
   cache: false
 });
 
-
+var errorDisplay = function (url, status, errorMessage) {
+  $('#debugPanel').html('<h3>' + status + '</h3><p><code>' + url + '</code></p><p>' + errorMessage + '</p>');
+  $('#debugPanel').fadeIn().delay(5000).fadeOut();
+};
      
```

Comment:
Is "debugPanel" used for things other than error messages?  Should there be a panel dedicated to errors?